dist/send_barcode.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+"use strict";
+let amqp = require('amqplib/callback_api');
+amqp.connect(`amqp://${process.env.RABBITMQ_USER}:${process.env.RABBITMQ_USER_PW}@${process.env.RABBITMQ_DEV_HOST}`, function (error0, connection) {
```

Comment:
Why is this not a ts file? 
also use strict is defined in the ts config doesn't need to be included here.

Reply:
This is just a test file that I use to test quickly if the sending works.

Reply:
Okay but we don't push js files

Reply:
@Darjusch we don't push js-files?
<img width="150" alt="image" src="https://user-images.githubusercontent.com/23193392/112764648-da822600-9009-11eb-945a-7fac3008a574.png">

We sad we don't want to push js-files which are getting generated by tsc anyhow, so the PR getting cleaner. ;)

Reply:
exactly and you pushed the send_ barcode.js and send_barcode.ts